contracts/strategies/StrategyCurveBTCVoterProxy.sol

```diff
@@ -76,18 +92,17 @@ contract StrategyCurveSBTC {
     function deposit() public {
         uint256 _want = IERC20(want).balanceOf(address(this));
         if (_want > 0) {
-            IERC20(want).safeApprove(pool, 0);
-            IERC20(want).safeApprove(pool, _want);
-            Gauge(pool).deposit(_want);
+            IERC20(want).safeTransfer(proxy, _want);
+            VoterProxy(proxy).deposit(gauge, want);
```

Comment:
Just so I understand this section:
1. We no longer need approvals because we are transfering
2. We are depositing through the VoterProxy instead of directly now